src/nni_manager/training_service/kubernetes/kubeflow/kubeflowTrainingService.ts

```diff
@@ -172,12 +180,9 @@ class KubeflowTrainingService extends KubernetesTrainingService implements Kuber
     }
 
     /**
-     * upload code files to nfs or azureStroage
-     * @param trialJobId
-     * @param trialLocalTempFolder
-     * return: trialJobOutputUrl
+     * upload local folder to nfs or azureStroage
      */
-    private async uploadCodeFiles(trialJobId: string, trialLocalTempFolder: string): Promise<string> {
+    private async uploadFolder(srcDirectory: string, destDirectory: string): Promise<string> {
```

Comment:
if the logic of this function is the same for framework controller, kubeflow, and PAI, why not extract it as a util function and put it into storage utils?

Reply:
The logic is different according to different storage types, for example `nfs`, `azureStorage`, `SSH connection`. The storage account or ssh connection information is set in related trainingService, extract it as a common function does not have obviously optimization.